app/src/main/java/com/kodeco/android/countryinfo/api/CountryAPIService.kt

```diff
@@ -0,0 +1,10 @@
+package com.kodeco.android.countryinfo.api
+
+import com.kodeco.android.countryinfo.models.Country
+import retrofit2.Response
+import retrofit2.http.GET
+
+interface CountryAPIService {
+    @GET("all")
+    suspend fun getAllCountries(): Response<List<Country>>
```

Comment:
nitpick but you could remove the `Response` wrapper here and just return `List<Country>` type.

Reply:
Implemented this change and pushed. Helped reduce the number of the code lines and much simpler. Thank you.